save-backend/src/main/resources/application-dev.properties

```diff
@@ -4,6 +4,7 @@ spring.datasource.url=jdbc:mysql://localhost:3306/save_cloud
 datasource.dev.url=jdbc:mysql://localhost:3306/save_cloud
 spring.datasource.username=root
 spring.datasource.password=123
+backend.url=http://localhost:${server.port}
```

Comment:
how we missed that? why it wasn't in the profile

Reply:
it's a new property, added in scope of this PR